src/mkdocstrings/handlers/python.py

```diff
@@ -45,6 +46,7 @@ class PythonRenderer(BaseRenderer):
         "show_if_no_docstring": False,
         "show_signature_annotations": False,
         "show_source": True,
+        "show_bases": True,
```

Comment:
I was going to ask to default to false as to not alter previous output, but I guess showing base classes is the sane default option, since it doesn't break anything and is nice to see. If you have an opinion on this please share!

Reply:
You want to know what the base classes are more often than not, since it tells where you can use it (anything that takes a `BaseRenderer` can take a `PythonRenderer`, since it's an interface), and what additional methods are available.  You could turn on `show_inherited_members`, but this brings in a bunch of noise from builtin Python classes.

Sometimes a base class is an implementation detail, but that's rarer.

Reply:
I'll leave it on by default unless you have strong objections.